readthedocs/core/models.py

```diff
@@ -27,10 +28,6 @@ def __unicode__(self):
         return (ugettext("%(username)s's profile")
                 % {'username': self.user.username})
 
-    def get_form(self):
```

Comment:
The `get_form` *is* used as far as I can tell: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/templates/profiles/public/profile_detail.html#L11
However, the `form` variable is then again not used in this template, and not in the included `project_list_detailed.html` :) So we only need to remove the usage of `get_form` from the template and should be set.